extensions/cl_extension_template.asciidoc

```diff
@@ -389,7 +379,7 @@ to the other extension specs, then this section may be omitted.
 things?
 +
 --
-+RESOLVED+: Only interesting things.  Non-interesting things should
+`RESOLVED`: Only interesting things.  Non-interesting things should
```

Comment:
This is inconsistent with the syntax used for UNRESOLVED below. FWIW, I think I prefer bold for these.